infra/src/main/kotlin/org/yapp/infra/aop/aspect/ServiceLoggingAspect.kt

```
@@ -0,0 +1,121 @@
+    private fun maskMapLikeObject(obj: Any): String {
+        return try {
+            val map: Map<*, *> = objectMapper.convertValue(obj, Map::class.java)
+            val maskedMap = map.mapValues { (key, value) ->
+                val keyStr = key.toString()
+                if (isSensitiveParameter(keyStr)) {
+                    return@mapValues MASKING_TEXT
+                }
+                value
+            }
+            objectMapper.writeValueAsString(maskedMap)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)